docs/release-notes/release-notes-0.19.0.md

```diff
@@ -74,6 +74,10 @@
   This is a protocol gadget required for Dynamic Commitments and Splicing that
   will be added later.
 
+* [Fixed](https://github.com/lightningnetwork/lnd/pull/9424) a case where the
```

Comment:
so basically the historical syncer might get hang up in a race condition not being able to serve nor request any historical graph syncs ?

Reply:
yeah unless the remote sends another query reply back, which it won't. Usually a reconnect can fix it, but cannot guarantee it succeeds.